src/hyperfine/app.rs

```diff
@@ -129,6 +129,21 @@ fn build_app() -> App<'static, 'static> {
                      This performs benchmarks for 'sleep 0.3', 'sleep 0.5' and 'sleep 0.7'.",
                 ),
         )
+        .arg(
+            Arg::with_name("parameter-list")
+                .long("parameter-list")
+                .short("L")
+                .takes_value(true)
+                .allow_hyphen_values(true)
+                .value_names(&["VAR", "VALUES"])
+                .conflicts_with("parameter-scan")
+                .help(
+                    "Perform benchmark runs for each value in the comma-seperated list VALUES. \
```

Comment:
What do you think: we could rename `VALUES` to `LIST`. This way, we could write: *"Perform benchmark runs for each value in the comma-seperated LIST"*. Not sure if that's necessarily better/clearer.